tools/abel/validators/README.md

```diff
@@ -0,0 +1,197 @@
+
+Below is a definition of each table and column in the ABEL spreadsheet:
+- [Site](#site)
+- [Entities](#entities)
+- [Entity Fields](#entity-fields)
+- [States](#states)
+- [Connections](#connections)
+
+### Site
+
+The `Site` tab holds all information about a project site or building.
+
+`Building Code` *string* **required**
+
+A [human-readable code](../../../ontology/docs/building_config.md#identifiers) for a building. e.g. `US-SVL-TC2`
+
+`Entity Guid` *string*
+
+[UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)) [identifier](../../../ontology/docs/building_config.md#identifiers) for a building.
+Unless you are onboarding a Google building, this value is generated by ABEL and does not need to be provided for initial onboarding.
+
+***Note:*** A Site should not be considered an entity and should not be listed in
+the `Entities` tab.
+
+### Entities
+
+The `Entities` tab holds all information about [reporting entities](../../../ontology/docs/building_config.md#reporting-physical-devices) and [virtual entities](../../../ontology/docs/building_config.md#virtual-devices) in a building or site. If ABEL is being used for Google then Guids are required for all entities in the `FACILTITIES` namespace which can be retrieved from DB API.
+
+`Entity Code` *string* **required**
+
+[Human-readable code](../../../ontology/docs/building_config.md#identifiers) for an entity e.g. `AHU-1001`
+
+`Entity Guid` *string*
+
+[UUID4](https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random))
+[identifier](../../../ontology/docs/building_config.md#identifiers) for an entity. **Note:** If an entity belongs to the `FACITITLIES` namespace and ABEL is
+being used or Google, then this field is **required** or else spreadsheet
+validation will fail.
+
+`Etag` *string*
+
+Version control tag for entities in an entity type database. Used only when updating an existing building config.
+
+`Is Reporting` *Boolean* **required**
+
+`TRUE` if an entity is [reporting data](../../../ontology/docs/building_config.md#reporting-physical-devices) and `FALSE` if [the entity is virtual](../../../ontology/docs/building_config.md#virtual-devices).
+
+`Cloud Device ID` *string*
+
+Cloud registry [device numeric identifier](../../../ontology/docs/building_config.md#identifiers). The device numeric ID is automatically created by a Cloud IoT Core (or similar cloud iot service), and it is globally unique and not editable.
+
+`DBO Namespace` *string* **required**
+
+[Namespace](../../../ontology/docs/ontology.md#namespaces) for an entity as defined in the [Digital Buidings Ontology](../../../ontology/README.md). e.g. `HVAC`
+
+`DBO General Type` *string*
+
+[General Type](../../../ontology/docs/model.md#general-types) for an entity as defined in the [Digital Buildings Ontology](../../../ontology/README.md). e.g. `AHU`
```

Comment:
as defined in *a particular namespace* in the DBO.

Reply:
Done.